autotx/agents/SwapTokensAgent.py

```diff
@@ -73,7 +73,7 @@ def _run(self, exact_amount_in: float, token_in: str, token_out: str) -> str:
             token_in_address,
             token_out_address,
             self.recipient.hex,
-            is_exact_input=True,
```

Comment:
I think it's clearer if the arg is named

Reply:
me too but python doesn't like it :sob:

Reply:
maybe we could define the variable and pass it?